runner/steps.ts

```diff
@@ -161,7 +161,9 @@ function runBrowsers(context: Context) {
       // TODO(nevir): Better rationalize run-end and hook.
       await context.emitHook('cleanup');
 
-      return error;
+      if (error) {
+        throw error;
```

Comment:
Is throwing an Error more useful here? It won't have a particularly insightful stack trace.